apps/src/templates/studioHomepages/TeacherHomepage.jsx

```diff
@@ -111,7 +111,7 @@ export default class TeacherHomepage extends Component {
 
   componentDidMount() {
     // The component used here is implemented in legacy HAML/CSS rather than React.
-    $('#terms_reminder').appendTo(ReactDOM.findDOMNode(this.refs.termsReminder)).show();
```

Comment:
I tried renaming the ref to more accurately describe the things it shows, but I couldn't rename `#terms_reminder`. When I renamed it (on both this .jsx page and in the _homepage.html.haml page), the layout would go wonky and the reminders would be placed on top of the banner. I couldn't find any css styles attached to this ID so I'm not sure why only `#terms_reminder` works to show the views below the banner.

Reply:
I'm seeing the wonkiness on this PR as well:

![screenshot 2018-05-12 17 35 19](https://user-images.githubusercontent.com/2205926/39962669-e13899f2-560a-11e8-81ac-893563e37fd1.png)

The trick is that we don't want the version in the HAML to be visible; it should be wrapped in something marked `display: none`.  Then, the HTML is copied from there and appended to the React DOM, in this `appendTo` line.  Once we're showing it in the React DOM, then it should appear below the React that contains the "My Dashboard" heading and provides the appropriate spacing.

Note that `#terms_reminder` was the parent div for a variety of components, not just this one, so it was responsible for making several things available in React.  It's likely that this could get the `display: none` treatment so that none of the components under it, if they were being rendered, would be shown, assuming that all should now be in the React DOM.

Reply:
@Erin007 since you [added in the terms reminder here](https://github.com/code-dot-org/code-dot-org/pull/15492), can you comment on what I'm missing here to fix my PR? :) I also don't understand why I would see my layout just fine but Brendan would see it wonky for the same PR...

Reply:
So I pulled the latest changes and it looks okay now.  The survey being shown is part of the React DOM which is how it should work.

Reply:
I was also able to do the rename just fine.  You have to make sure apps rebuilds successfully for that rename to work on the React side.